app/assets/javascripts/leaflet.layers.js

```diff
@@ -176,6 +176,7 @@ L.OSM.layers = function(options) {
 
       addOverlay(map.noteLayer, 'notes', OSM.MAX_NOTE_REQUEST_AREA);
       addOverlay(map.dataLayer, 'data', OSM.MAX_REQUEST_AREA);
+      addOverlay(map.gpsLayer, 'gps', OSM.MAX_REQUEST_AREA);
```

Comment:
We either need to pass a larger value for the second argument here (because I think the GPS tiles don't have a limit) or we need to add `javascripts.site.map_gps_zoom_in_tooltip` to the resources.